### ISSUE TITLE
Can't use only cse, redirect and 3ds2 of 4.6.0 SDK

### ISSUE THREAD

**Issue by zsoltvilagos** [Adyen/adyen-android#706]
**Describe the bug**

Can't use only `cse`, `redirect` and `3ds2` components of Adyen Android SDK 4.6.0 because `ui-core` requires a style from `card` library.

**To Reproduce**
1. Create a project with these dependencies:
    * com.adyen.checkout:cse:4.6.0
    * com.adyen.checkout:redirect:4.6.0
    * com.adyen.checkout:3ds2:4.6.0
 
3. Build

**Expected behavior**

It builds.

**Screenshots**

Build fails:
```
:app:processDebugResources
simple_text_item_view.xml
Android resource linking failed

> Task :app:processDebugResources FAILED
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"/Users/vilagoszs/.gradle/caches/transforms-3/5613a3be06b3f27a3802434af49ff65b/transformed/ui-core-4.6.0/res/layout/simple_text_item_view.xml","position":{"startLine":15}}],"original":"ERROR:/Users/vilagoszs/.gradle/caches/transforms-3/5613a3be06b3f27a3802434af49ff65b/transformed/ui-core-4.6.0/res/layout/simple_text_item_view.xml:16: AAPT: error: resource style/AdyenCheckout.InstallmentOptionTextView (aka <APPLICATION-ID>:style/AdyenCheckout.InstallmentOptionTextView) not found.\n    ","tool":"AAPT"}
```

**Smartphone (please complete the following information):**
 - Device: -
 - OS: -
 - Browser -
 - Version -

**Additional context**

https://github.com/Adyen/adyen-android/blob/4.6.0/ui-core/src/main/res/layout/simple_text_item_view.xml#L18 of `ui-core` has:

```
style="@style/AdyenCheckout.InstallmentOptionTextView"
```

this style is in `card` component: https://github.com/Adyen/adyen-android/blob/4.6.0/card/src/main/res/values/styles.xml#L147

As far as I see, out of `cse`, `redirect` and `3ds2` modules, only `redirect` requires `ui-core` (directly or transitively).

Furthermore, seems to me that only https://github.com/Adyen/adyen-android/blob/4.6.0/ui-core/src/main/java/com/adyen/checkout/components/ui/adapter/SimpleTextListAdapter.kt (`ui-core`!) uses `simple_text_item_view.xml` which in turn is only used by https://github.com/Adyen/adyen-android/blob/4.6.0/card/src/main/java/com/adyen/checkout/card/ui/AddressFormInput.kt (`card`!).

Maybe move layout and  `SimpleTextListAdapter` to `card` component?

**Comment by jreij** [comment]
Hi, thanks for reaching out and for the detailed investigation! I believe you got the issue spot on 👍  We'll let you know when we have a fix.

**Comment by jreij** [comment]
We just released [4.6.1](https://github.com/Adyen/adyen-android/releases/tag/4.6.1) with a fix to this issue 👍

**Comment by moia-zsolt** [comment]
> We just released [4.6.1](https://github.com/Adyen/adyen-android/releases/tag/4.6.1) with a fix to this issue 👍

Thanks a lot for the fast reaction.
